### PR TITLE
Feature/show score

### PR DESCRIPTION
**Pull request by gauthierboeshertz**
Display the score on the map interface

### CODE REVIEW

**Review by Alvaro2112 — APPROVED** [review]

File: app/src/main/java/sdp/moneyrun/map/MapActivity.java [hunk 1]
```diff
@@ -233,6 +234,8 @@ public void removeCoin(Coin coin) {
         }
         remainingCoins.remove(coin);
         collectedCoins.add(coin);
+        currentScore += coin.getValue();
+        currentScoreView.setText("Score: "+Integer.toString(currentScore));
```
Comment (ArnaudPoletto) [inline app/src/main/java/sdp/moneyrun/map/MapActivity.java:238]: It would be better to use a resource string.

**Review by ArnaudPoletto — CHANGES_REQUESTED** [review]
All good, just try to use resource string so we do not need to change that later

File: app/src/main/java/sdp/moneyrun/map/MapActivity.java (hunk 1 above)
Comment (gauthierboeshertz) [reply app/src/main/java/sdp/moneyrun/map/MapActivity.java:238]: Can we use a resource string for something that changes ?  the textview  is changed every time the player grabs a coin

Comment (ArnaudPoletto) [reply app/src/main/java/sdp/moneyrun/map/MapActivity.java:238]: You could define a new string resource like this

Comment (gauthierboeshertz) [reply app/src/main/java/sdp/moneyrun/map/MapActivity.java]: Ok I have changed the string resource

**Review by ArnaudPoletto — APPROVED** [review]

**Pull request merged**